newdle/client/src/client.js

```diff
@@ -32,15 +33,33 @@ class Client {
     return this._request(flask`api.users`({q}));
   }
 
-  async _request(url, withStatus = false, isRetry = false) {
-    const headers = {Accept: 'application/json'};
+  createNewdle() {
+    return this._request(flask`api.create_newdle`(), {
+      method: 'POST',
+      body: JSON.stringify({
+        title: getTitle(this.store.getState()),
```

Comment:
the client should not access the store - the only reason it has access to it is for the token refresh logic. regular client function should receive everything as params instead.